### DISCUSSION TITLE
Internal and short link domains for multiple hubs installations

### DISCUSSION THREAD

**Discussion by smakhtin** [mozilla/hubs#3120] [Hubs Cloud Questions] [answered]
Hi, I need to have 2 hubs instances on a single AWS account, both are using the same domain, via different subdomains. Do I need different internal domains and short link domains for them? Or I can somehow set up short link and internal subdomains as well?

**Comment by robinkwilson** [comment, marked as answer]
You're welcome to try with explicit Hosted Zone subdomains! But I haven't heard of any successful deploys setups that way (yet!). 

Relevant feature requests/issues: 
https://github.com/mozilla/hubs-cloud/issues/100 
https://github.com/mozilla/hubs-cloud/issues/80 
https://github.com/mozilla/hubs/discussions/2874

**Reply by smakhtin** [reply]
@robin-k-wilson any docs on explicit Hosted Zone subdomains? I checked issues — experiencing similar problems.
